cobra/core/configuration.py

```diff
@@ -50,8 +52,10 @@ class BaseConfiguration(object):
 
     def __init__(self):
         self._solver = None
+        self.tolerance = 1E-09
```

Comment:
As @cdiener suggested, I will check feasibility and blocked reactions for all models that I have available using 1E-09 and 1E-06 as tolerance values.